src/libraries/Microsoft.PowerFx.Core/Binding/BindInfo/FirstNameInfo.cs

```diff
@@ -18,6 +18,8 @@ internal sealed class FirstNameInfo : NameInfo
     {
         public override DName Name => Node.AsFirstName().Ident.Name;
 
+        public DName DisplayName { get; private set; }
```

Comment:
>private set [](http://example.com/codeflow?start=40&length=11)

`init`?

Reply:
Also, is this always the display name? under what circumstances is it set? Add a comment clarifying that.

Reply:
We mainly set it here 

                if (maybeFunc.HasIdentifiers && _features.HasFlag(Features.SupportIdentifiers))
                {
                    var i = 0;

                    foreach (var arg in args)
                    {
                        i++;

                        if (arg is FirstNameNode firstNameNode && maybeFunc.IsIdentifierParam(i))
                        {
                            var displayName = GetLogicalNodeNameAndUpdateDisplayNames(argTypes[0], firstNameNode.Ident, out _);

                            if (argTypes[0].TryGetType(displayName, out var fieldType))
                            {
                                var fileNameInfo = FirstNameInfo.Create(firstNameNode, new NameLookupInfo(BindKind.NamedValue, fieldType, DPath.Root, 0, displayName: new DName(displayName)));
                                _txb.SetInfo(firstNameNode, fileNameInfo);
                            }
                        }
                    }
                }